app/code/Magento/Catalog/Controller/Adminhtml/Product/AddAttributeToTemplate.php

```diff
@@ -206,10 +236,6 @@ private function getBasicAttributeSearchCriteriaBuilder()
      */
     private function getAttributeRepository()
```

Comment:
Please, remove method and fix all usages to `$this->attributeRepository`

Reply:
Fixed, thanks.